### PR TITLE
[tools/onert_train] Add onert_train usage example to readme

### PR DESCRIPTION
**Pull request by zetwhite**
This PR adds a example to use onert_train in the readme.

ONE-DCO-1.0-Signed-off-by: SeungHui Youn <sseung.youn@samsung.com>

### CODE REVIEW

File: tests/tools/onert_train/README.md [hunk 1]
```diff
@@ -34,3 +34,62 @@ onert_train \
 ```
 
 `onert_train --help` would help you to set each parameter.
+
+## Example
```
Comment (zetwhite) [inline tests/tools/onert_train/README.md:38]: This example is for ONE developers, but who are new to the training feature.
I thought that `usage` part isn't enough for the newbie. So I tried to explain a bit more.

But I'm not sure that it is meaningful, or easy to maintain later. So, Any opinion is welcome.

File: tests/tools/onert_train/README.md [hunk 2]
```diff
@@ -34,3 +34,61 @@ onert_train \
...
+# check data files are generated
+# There are 'mnist.train.input.1000.bin' and 'mnist.train.output.1000.bin'
+$ tree out
+```
+
+### Prepare a circle model file
+
+`onert_train` use a `*.circle` file or a nnpackage as input. <br/>
+
+<!-- This readme is for the ONE developers, so they might know the onecc usage.-->
```
Comment (YongseopKim) [inline tests/tools/onert_train/README.md:72]: How about adding content to link to `onecc` usage?

**Review by YongseopKim — APPROVED** [review]
LGTM 👍

File: tests/tools/onert_train/README.md (hunk 2 above)
Comment (zetwhite) [reply tests/tools/onert_train/README.md:72]: I'd love to. 
But is there a `onecc` usage document in public? I tried to find but I couldn't fine any document.

**Review by jyoungyun — APPROVED** [review]
LGTM

**Pull request merged**